src/bundle/Resources/public/scss/_icons.scss

```diff
@@ -7,11 +7,26 @@
         height: calculateRem(16px);
     }
 
+    &--small-medium {
+        width: calculateRem(20px);
+        height: calculateRem(20px);
+    }
+
     &--medium {
         width: calculateRem(24px);
         height: calculateRem(24px);
     }
 
+    &--large {
```

Comment:
I wonder if `48px` and `64px` are based on elements from the interface?

Reply:
Yes, these sizes come from AdminUI modules.

Reply:
Great!